src/components/LandingNavbar.jsx

```diff
@@ -36,11 +34,25 @@ function LandingNavbar() {
           </Nav>
         </Navbar.Collapse>
       </div>
-      <div className="order-3 ml-auto">
-        <Btn title="Sign in with SSO" type="primary" onClick={() => login(history)} />
+      <div className="order-3 sso">
+        <SSO />
       </div>
     </Navbar>
   );
 }
 
+function SSO() {
+  const history = useHistory();
+  const pathName = useLocation()["pathname"];
```

Comment:
you can use [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) here